version/version.go

```diff
@@ -0,0 +1,17 @@
+package version
+
+import "github.com/hashicorp/packer-plugin-sdk/version"
+
+var (
+	// Version is the main version number that is being run at the moment.
+	Version = "0.0.1"
```

Comment:
Should this be ?
```suggestion
	Version = "0.0.2"
```
for the next version?